File: Python/produk.py
class ProdukElektronik:
    def __init__(self, id, nama, merk, harga, stok):
        self.id = id
        self.nama = nama
        self.merk = merk
        self.harga = harga
        self.stok = stok

    def tampilkan(self):
        print(f"ID: {self.id}, Nama: {self.nama}, Merk: {self.merk}, Harga: {self.harga}, Stok: {self.stok}")

File: CPP/produk.cpp
#include <iostream>
using namespace std;

class ProdukElektronik {
public:
    string id, nama, merk;
    int harga, stok;

    ProdukElektronik(string i, string n, string m, int h, int s) {
        id = i; nama = n; merk = m; harga = h; stok = s;
    }

    void tampilkan() {
        cout << "ID: " << id << ", Nama: " << nama << ", Merk: " << merk
             << ", Harga: " << harga << ", Stok: " << stok << endl;
    }
};
